home/ivp-planning/advanced-tools/markup-tool.md

```diff
@@ -0,0 +1,103 @@
+For better understanding and training we created a ![](../../../.gitbook/assets/YouTube\_icon.png) <img src="../.gitbook/assets/YouTube_icon.png" alt="" data-size="line"> [iVP YouTube Channel.](https://www.youtube.com/watch?v=DYKIYcZMrVE&list=PLlzoGkRUR67houzn5F5ejD3R-kQrDcps5&index=16) where you can find a detailed tutorial for the markup tool.
+{% endhint %}
+
+## Placement:
+
+The markup tool can be activated by clicking the markup icon in the [2D-panel toolbar](../user-interface/the-2d-panel.md#the-toolbar-of-the-2d-panel).
+
+![](../../../.gitbook/assets/markup\_icon.jpg)
+
+As soon as the tool is active, the icon will turn a small markup icon will follow the mouse cursor to remind you that you are in the markup mode.
+
+![](../../../.gitbook/assets/markup\_cursor.jpg)
+
+To create a markup, **left click on a position** in the [2D-panel](../user-interface/the-2d-panel.md) or [3D-panel](../user-interface/the-3d-panel.md). A blue markup (a pin resembling a map location marker) will appear that marks the spot where you clicked and functions as an interactable highlight point.
+
+![](../../../.gitbook/assets/markup\_placement.jpg)
+
+The placed markup will be added to the [hierarchy](../user-interface/the-machine-list.md) and can be used as any other object. It can be [renamed and grouped in folders](../the-tree-view/renaming-objects-and-folders.md), [set in/visible](../the-tree-view/visibility-of-objects-and-folders.md), set to be [un/locked](../the-tree-view/locked-objects-and-folders.md) and [recolored](../machines/highlighting-objects.md).
+
+## Editing a Markup:
+
+After placing a markup, the [properties panel](../user-interface/the-info-panel.md) will show the default options to [move an object](../machines/move-objects#moving-objects-via-the-info-panel.md) and most foremost a section to edit the selected markup.
+
+![](../../../.gitbook/assets/markup\_properties.jpg)
+
+## The default fields:
+
+* Default: the general information of a markup regarding
+    * _**Name:**_ the name of the object which [can be changed](../the-tree-view/renaming-objects-and-folders.md) at any time
+    * _**ID:**_ the unique ID of the object which is generated when the markup is [placed](../machines/first-steps-with-3d-object.md) in the [2D-](../user-interface/the-2d-panel.md) or [3D-panel](../user-interface/the-3d-panel.md); the ID is unchangeable 
+    * _**Author:**_ the author of the markup, per default "Planner"; this information is unchangeable and independent from the object name
+    * _**Created:**_ exact time and date when the markup was created; this information is unchangeable
+* Transform: the panel containing information of the markup regarding
+    * _**Dimensions:**_ the dimensions of the object; this information is only changeable by changing the [scale](../machines/scale-objects.md)
+    * _**Position:**_ the position of the object, changing this will [move the object](../machines/move-objects#moving-objects-via-the-info-panel.md)
```

Comment:
The file ending needs to be before the `#` for the anchor to work

Reply:
changed.